### ISSUE TITLE
Extra DIV tag at end of message body

### ISSUE THREAD

**Issue by severdia** [Kunena/Kunena-Forum#3761]
#### Steps to reproduce the issue

View any topic page like this one:
https://www.playshakespeare.com/forum/as-you-like-it/127-variations-on-romantic-love
#### Expected result

The message body DIV should should completely before the table cell.
#### Actual result

There is an extra DIV.

```
<td class="kmessage-left">
<div class="kmsgbody">
<div class="kmsgtext">Lorem ipsum</div>
</div>
<div>
</td>
```
#### System information (as much as possible)

Joomla version: 3.5
Kunena version: 4.0.10
Php version: 5.6
Database version:

**Comment by 810** [comment]
i don't see any extra div tag. 

Only an empty div start and closing div.

```
<td class="kmessage-left" abp="164">

<!-- START components/com_kunena/template/blue_eagle/html/topic/default_message.php -->
<div class="kmsgbody" abp="165">
    <div class="kmsgtext" abp="166">
        Lorem ipsum </div>
</div>

<div class="kreply-form" id="kreply37_form" style="display: none;" abp="167">
    <form name="postform" action="/develop/forum" enctype="multipart/form-data" method="post" abp="168">
        <input name="view" type="hidden" value="topic" abp="169">
        <input name="task" type="hidden" value="post" abp="170">
        <input name="parentid" type="hidden" value="37" abp="171">
        <input name="catid" type="hidden" value="2" abp="172">
        <input name="e33a82bb0c77466a268f02684bc27d9a" type="hidden" value="1" abp="173">
                <input name="authorname" type="hidden" value="admin" abp="174">
                        <input name="subject" class="inputbox" type="text" size="35" maxlength="10" value="ss" abp="175"><br abp="176">
        <textarea name="message" class="inputbox" rows="6" cols="60" abp="177"></textarea><br abp="178">
                        <input name="submit" title="Click here to submit your message" class="kbutton kreply-submit" type="submit" value="Submit" abp="179">
        <input name="cancel" title="Click here to cancel your message" class="kbutton kreply-cancel" type="reset" value="Cancel" abp="180">
        <small abp="181">Note: BBcode and smileys are still usable.</small>
    </form>
</div>
<!-- END components/com_kunena/template/blue_eagle/html/topic/default_message.php -->
</td>
```

i have removed it here: https://github.com/Kunena/Kunena-Forum/pull/3762/files

**Comment by 810** [comment]
fixed on #3762
